js/foundation.offcanvas.js

```diff
@@ -154,8 +154,7 @@ class OffCanvas {
    */
   open(event, trigger) {
     if (this.$element.hasClass('is-open') || this.isRevealed) { return; }
-    var _this = this,
-        $body = $(document.body);
+    var _this = this
```

Comment:
Do we no longer need $body?  In which case we should update with semicolon after this. Otherwise we should put this dropped line back in.